projects/canopy/src/lib/forms/date/docs/guide.stories.mdx

```diff
@@ -47,130 +54,127 @@ and in your HTML:
-|------|-------------|:----:|:-----:|:-----:|
-| \`\`dateId\`\` | HTML ID attribute for the date input, auto generated if not provided | string | 'lg-input-date-\${nextUniqueId++}' | No |
-| \`\`monthId\`\` | HTML ID attribute for the month input, auto generated if not provided | string | 'lg-input-month-\${nextUniqueId++}' | No |
-| \`\`yearId\`\` | HTML ID attribute for the year input, auto generated if not provided | string | 'lg-input-year-\${nextUniqueId++}' | No |
-| \`\`ariaDescribedBy\`\` | HTML ID for the corresponding element that describes the date field, if not provided it will use the hint field where appropriate | string | null | No |
-| \`\`focus\`\` | Set the focus on the fieldset | boolean | null | No |
+### LgDateFieldComponent inputs
+| Name              | Description                                                                                                                       |   Type    |              Default               | Required |
+|-------------------|-----------------------------------------------------------------------------------------------------------------------------------|:---------:|:----------------------------------:|:--------:|
+| `dateId`          | HTML ID attribute for the date input, auto generated if not provided                                                              | `string`  | `lg-input-date-${nextUniqueId++}`  |    No    |
+| `monthId`         | HTML ID attribute for the month input, auto generated if not provided                                                             | `string`  | `lg-input-month-${nextUniqueId++}` |    No    |
+| `yearId`          | HTML ID attribute for the year input, auto generated if not provided                                                              | `string`  | `lg-input-year-${nextUniqueId++}`  |    No    |
+| `ariaDescribedBy` | HTML ID for the corresponding element that describes the date field, if not provided it will use the hint field where appropriate | `string`  |               `null`               |    No    |
+| `focus`           | Set the focus on the fieldset                                                                                                     | `boolean` |               `null`               |    No    |
 
 ## Validation
 Date validation is quite complex with a range of different validation errors, where possible we have encapsulated as much of that complexity into the components as we can.
 
 Internally the date-field can determine if the entered date is a valid date. If additional validation is required such as checking for past or future dates this can be done with external validators.
 
-For the validation to work properly it's very important to include the \`ngForm\` on the form tag and for the submit button to be within the form.
+For the validation to work properly it's very important to include the `ngForm` on the form tag and for the submit button to be within the form.
```

Comment:
This is also written on line 59, is the duplication intentional?

Reply:
Removed from here and left it only on line 59.